Dockerfile

```diff
@@ -1,16 +1,16 @@
 # Careful, the version and build date are both dates, but different formats
-ARG ARCH_VERSION="20200908"
-ARG BUILD_DATE="2020/09/18"
+ARG ARCH_VERSION="base-20201129.0.10056"
```

Comment:
I supposed the comment above is no longer necessary now that the tag changed formats. It should probably just explain that this is used as a tag from [dockerhub](https://hub.docker.com/_/archlinux?tab=tags&page=1&ordering=last_updated)

Reply:
Updated, thanks for pointing that out.